.github/workflows/build-and-run.yml

```diff
@@ -215,3 +216,29 @@ jobs:
         run: sleep 1
       - name: Check accel-ppp stat
         run: accel-cmd show stat
+
+  Build-in-Container-Alpine:
+    runs-on: ubuntu-latest
+    container:
+      image: alpine:3.19
```

Comment:
maybe `image: alpine:latest` ?

Reply:
This might have unintended effect if they upgrade distro, and new one might break builds (and it wont be related to PR), but generally i dont mind, will update.